source/components/ReadMoreReact.jsx

```diff
@@ -8,7 +8,7 @@ export default class ReadMoreReact extends React.Component {
 		this.state = { displaySecondary: false, primaryText: "", secondaryText: "", readMoreText: this.props.readMoreText };
 	}
 
-	componentDidMount() {
```

Comment:
ComponentWillMount is being deprecated, I can put it in the constructor but want to verify if that is the right path forward:
https://github.com/styled-components/styled-components/issues/1575